web/testing/promise-api/index.html

```diff
@@ -53,8 +57,18 @@
   <body>
     <h2>KeymanWeb Sample Page - Promise API</h2>
     <p>This page is for testing the Promises returned from adding keyboards.
-	  See <a href="https://github.com/keymanapp/keyman/issues/5044">Specification #5044</a>
+      See <a href="https://github.com/keymanapp/keyman/issues/5044">Specification #5044</a>
     </p>
+
+    <p id="modeSelect">You are currently testing under the alerts <em id="mode"> </em> mode.
+    Two alert modes are available:
+    <a id="alert_disabled" href="./index.html?useAlerts=false">Disabled</a> or
+    <a id="alert_default" href="./index.html">Enabled (Default)</a></p>
```

Comment:
A bit more context would be nice here.  It was a bit unintuitive to see native-browser alerts with alerts 'disabled', given the description.   Something like:

> This will only affect whether or not KeymanWeb's alert modal is displayed.  A native browser alert and dev-console error log will occur regardless of this setting when stub fetch errors occur.

Also, might not hurt to explicitly mention some good test examples / instructions, like:

> This test page is best used with Developer mode, as output for both successes and failures will be logged to the JS console.
> Visible alert-style feedback will only occur for invalid search terms, such as using "foobar" for any of the three keyboard-adding options at the bottom of the page.

While we understand this page's use right now, we may not easily remember it otherwise months down the line.